src/commands/service/create.ts

```diff
@@ -67,6 +75,31 @@ export const fetchValues = (opts: CommandOptions): CommandValues => {
     middlewaresArray = opts.middlewares.split(",").map((str) => str.trim());
   }
 
+  let serviceVgArray: string[] = [];
+  if (opts.serviceBuildVg && opts.serviceBuildVg.trim()) {
+    serviceVgArray = opts.serviceBuildVg.split(",").map((str) => str.trim());
+  }
+
+  let serviceVariablesArray: string[] = [];
+  if (opts.serviceBuildVariables && opts.serviceBuildVariables.trim()) {
+    serviceVariablesArray = opts.serviceBuildVariables
+      .split(",")
+      .map((str) => str.trim());
+  }
+
+  serviceVariablesArray.forEach(function (r) {
```

Comment:
Should use arrow functions when passing in lambdas: `(r) => {` instead of `function (r) {`

also, this isn't a required change, but `for...of` loops should be preferred over `.forEach`: `for (const r of serviceVariablesArray) {` instead of `serviceVariablesArray.forEach(function (r) {`

Reply:
Will do! Thanks!